src/Command/VotingPlatformInitializeCommitteeElectionCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\CommitteeElection;
+use AppBundle\Entity\VotingPlatform\Designation\Designation;
+use AppBundle\Repository\CommitteeRepository;
+use AppBundle\Repository\VotingPlatform\DesignationRepository;
+use AppBundle\VotingPlatform\Designation\DesignationTypeEnum;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class VotingPlatformInitializeCommitteeElectionCommand extends Command
+{
+    protected static $defaultName = 'app:voting-platform:initialize-committee-election';
+
+    /** @var DesignationRepository */
+    private $designationRepository;
+    /** @var SymfonyStyle */
+    private $io;
+    /** @var EntityManagerInterface */
+    private $entityManager;
+    /** @var CommitteeRepository */
+    private $committeeRepository;
+
+    protected function configure()
+    {
+        $this->setDescription('Configure Voting Platform');
```

Comment:
it is the same description of the other command, is it normal ?

Reply:
fixed, thanks :+1: